libs/cli/src/main/java/org/elasticsearch/cli/Terminal.java

```diff
@@ -259,7 +270,7 @@ public String readText(String text) {
             try {
                 final String line = getReader().readLine();
                 if (line == null) {
-                    throw new IllegalStateException("unable to read from standard input; is standard input open and a tty attached?");
+                    throw EMPTY_LINE_EXCEPTION;
```

Comment:
I don't think it's right to call this an `EMPTY_LINE_EXCEPTION`; the `readLine()` method returns an empty string for an empty line. It returns `null` when the end of the stream has been reached without reading any characters.